R/osmdata.R

```diff
@@ -333,11 +333,16 @@ get_osm_river <- function(bb, river_name, crs = NULL, force_download = FALSE) {
 #'
 #' # Ensure that data is not retrieved from cache
 #' get_osm_streets(bb, crs, force_download = TRUE)
+#' @srrstats {G2.16} This function checks numeric arguments for undefined values
+#'   (NaN, Inf, -Inf) and errors when encountering such values.
 get_osm_streets <- function(aoi, crs = NULL, highway_values = NULL,
                             force_download = FALSE) {
   # Check input
   checkmate::assert_true(inherits(aoi, c("sf", "sfc", "bbox")))
-  checkmate::assert_numeric(crs, null.ok = TRUE)
+  checkmate::assert_numeric(crs,
+                            null.ok = TRUE,
+                            any.missing = FALSE,
+                            finite = TRUE)
```

Comment:
This check feels weird - and I think will be superseded by #298 and #269 
```suggestion
  checkmate::assert_numeric(crs, null.ok = TRUE)
```